### PR TITLE
update submission procedure

### PR DESCRIPTION
**Pull request by AMashoshyna**
related to issue #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,15 +5,24 @@ This repo was created for students to submit their homeworks for review.
 
 ### How to submit
 - fork this repo
+- clone the your fork to your local machine
```
Comment (OleksiyRudenko) [inline README.md]:
```suggestion
- clone your fork to your local machine
```

File: README.md [hunk 2]
```diff
@@ -4,16 +4,26 @@
 This repo was created for students to submit their homeworks for review. 
 
 ### How to submit
-- fork this repo
-- in your fork, add folder with your github name under  `submissions` (if you haven't yet)
-- in your fork, under your name folder, add a folder with task name and put your code in the folder
+- fork this repository
+- clone the your fork to your local machine: 'git clone tream https://github.com/YOUR_NUSERNAME/frontend-2019-homeworks.git`
```
Comment (OleksiyRudenko) [inline README.md]: `tream` seems to be a typo.

File: README.md [hunk 3]
```diff
@@ -4,16 +4,26 @@
...
+- in your local repository, under your name folder, add a folder with task name and put your code in the folder
 
   See example file structure you are expected to have below:
 
   ![File structure example](img/file-structure.png)
 
-- make a pull-request to this repository to add your changes
-- assign course mentors to review your pull-request
+- make pull-request to this repository following these steps:
+  - create new branch, name it according to task performed (aka feature branch): `git checkout -b dom-api-task`. In this example feature branch is called `dom-api-task`
```
Comment (OleksiyRudenko) [inline README.md]:
```suggestion
  - `git checkout master` and then create new branch, name it according to task performed (aka feature branch): `git checkout -b dom-api-task`. In this example feature branch is called `dom-api-task`
```

Comment (OleksiyRudenko) [reply README.md]: Just to secure that students won't branch from other branches that are not merged into upstream master yet.

**Comment by AMashoshyna** [comment]
@OleksiyRudenko thanks!

**Pull request merged**